test/RouteCollectorTest.php

```diff
@@ -18,6 +19,7 @@
 use Psr\Http\Server\MiddlewareInterface;
 use Psr\Http\Server\RequestHandlerInterface;
 use TypeError;
+use function var_dump;
```

Comment:
Please move import function to function below classes - see line 28.

Reply:
fixed